oap-server/server-bootstrap/src/main/resources/application.yml

```diff
@@ -119,6 +122,9 @@ storage:
     # instead, give more query criteria (e.g. service id or time range), to narrow the query results.
     # see https://www.elastic.co/guide/en/elasticsearch/guide/current/pagination.html for more information
     indexMaxResultWindow: ${SW_STORAGE_ES_INDEX_MAX_RESULT_WINDOW:5000}
+#    advanced:
+#      index.translog.durability: "async"
+#      index.translog.sync_interval: "120s"
```

Comment:
According to the official document, `index.translog.sync_interval` default value is 5s,  `index.translog.durability` is `request`. But you set the different, are these two recommended by you? If so, it should be active configs in default. 

And you miss to update the documentation of storage.

Reply:
The default values are referring from https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules-translog.html#_translog_settings

Reply:
> According to the official document, `index.translog.sync_interval` default value is 5s, `index.translog.durability` is `request`. But you set the different, are these two recommended by you? If so, it should be active configs in default.
> 
> And you miss to update the documentation of storage.

same question here